zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -3341,6 +3341,72 @@ paths:
                 $ref: "#/components/schemas/ProblemDetail"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /resources/{resourceKey}:
+    get:
+      tags:
+        - Resource
+      operationId: getResource
```

Comment:
❓ The `operationId` is not in the original PR. Is this expected?

Reply:
Yes, this is only enforced for the spec on `main` to provide more consistent URLs in our generated public docs starting with 8.8. Otherwise, they depend on the summary of the path.